package.json

```diff
@@ -17,30 +17,35 @@
     "url": "https://raw.github.com/jeanlauliac/ninja-build-gen/master/LICENSE"
   },
   "author": "Jean Lauliac <jean@lauliac.com>",
-  "contributors": ["Tylor Reynolds"],
-  "files": ["lib"],
+  "contributors": [
+    "Tylor Reynolds"
+  ],
+  "files": [
+    "lib"
+  ],
   "main": "lib/ninja-build-gen.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/jeanlauliac/ninja-build-gen.git"
   },
   "scripts": {
-    "prepublish": "grunt",
+    "build": "grunt",
```

Comment:
So the problem is, prepublish seems to run when you do do `npm install`, so before `grunt-cli` is installed. I am not sure why it does that.